### ISSUE TITLE
[Question]: TS Command Removal?

### ISSUE THREAD

**Issue by rfid9** [danielmiessler/fabric#849]
### What is your question?

Did `ts` get removed from the project? Is there a plan to bring it back?

**Comment by rfid9** [comment]
> > go install github.com/danielmiessler/yt@latest

yt is the youtube tool. ts was used for local files.

**Comment by Nilirad** [comment]
I guess it was accidentally removed during the Go rewrite.

**Comment by ksylvan** [comment]
Yes. But I'm about to add a transcription method using Groq or OpenAI speech to text models.

**Comment by ksylvan** [comment]
https://github.com/danielmiessler/Fabric/issues/1711

**Comment by ksylvan** [comment]
https://www.linkedin.com/posts/kayvansylvan_release-release-v14291-danielmiessler-activity-7363261488100204544-yJt2

🎙️ Just shipped speech-to-text to Fabric CLI v1.4.291! 

Now you can transcribe audio and video files directly through the command line and pipe them into any of Fabric's AI patterns.

Key features:

• `--transcribe-file` to process audio/video files
• `--transcribe-model` with auto-completion for model selection  
• `--split-media-file` to handle large files (25MB+)
• Powered by OpenAI Whisper + GPT-4o Transcribe Models

This means you can now take a podcast, meeting recording, or video and instantly:

→ Extract insights with `extract_wisdom`
→ Summarize with `summarize` 
→ Create meeting notes with `create_meeting_notes`
→ And use any of the 200+ other patterns

The workflow is seamless - transcription happens automatically before processing, so you get clean, structured output from your audio content.

Another step toward making AI analysis accessible for any type of content 🚀

#AI #CLI #OpenSource #Productivity #AudioProcessing

https://lnkd.in/ghtXjDUx

Check out [Daniel Miessler 🛡️](https://www.linkedin.com/feed/update/urn:li:activity:7363261488100204544/#)’s Fabric. It’s the best, most up-to-date CLI tool for integrating AI tools into your everyday work. It saves me literally hours of work every day.
